client-app/pages/sign-up/sign-up-page.vue

```diff
@@ -1,56 +1,56 @@
 <template>
   <TwoColumn class="max-w-screen-xl">
     <template #left>
-      <h1 class="uppercase tracking-wide text-3xl lg:text-4xl font-bold mb-8 lg:mt-5">Registration</h1>
+      <h1 class="uppercase tracking-wide text-3xl lg:text-4xl font-bold mb-8 lg:mt-5" v-t="'pages.sign-up.header'"></h1>
```

Comment:
Why "sign-up" is dashed? All previous similar page keys were renamed to underscore notation.